src/XRP/model/xrp-payment-channel-create.ts

```diff
@@ -0,0 +1,77 @@
+import { PaymentChannelCreate } from '../Generated/web/org/xrpl/rpc/v1/transaction_pb'
+import { XRPCurrencyAmount } from '.'
+
+/*
+ * Represents a PaymentChannelCreate transaction on the XRP Ledger.
+
+ * A PaymentChannelCreate transaction creates a unidirectional channel and funds it with XRP.
+ * The address sending this transaction becomes the "source address" of the payment channel.
+ *
+ * @see: https://xrpl.org/paymentchannelcreate.html
+ */
+export default class XRPPaymentChannelCreate {
+  /**
+   * Constructs an XRPPaymentChannelCreate from a PaymentChannelCreate protocol buffer.
+   *
+   * @param paymentChannelCreate a PaymentChannelCreate (protobuf object) whose field values will be used to construct an XRPPaymentChannelCreate
```

Comment:
nit: add a period at the end

(It is the style that will be enforced by the linter eventually so we may as well start)